telethon_tests/test_higher_level.py

```diff
@@ -10,16 +10,17 @@
 api_id = None
 api_hash = None
 
-if not api_id or not api_hash:
-    raise ValueError('Please fill in both your api_id and api_hash.')
-
 
 class HigherLevelTests(unittest.TestCase):
-    @staticmethod
-    def test_cdn_download():
+    def setUp(self):
+        if not api_id or not api_hash:
+            raise ValueError('Please fill in both your api_id and api_hash.')
+
+    @unittest.skip("you can't seriously trash random mobile numbers like that :)")
```

Comment:
I missed this. What do you mean by that? They're test numbers and that's how you use them. Picking one randomly is the safest way to do it.

Reply:
Yeah that is wrong on my side. At this point of time I was somehow thinking numbers would be trashed this way. 

Currently I don‘t remember if the test passed though...